app/src/acceptance/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config xmlns:tools="http://schemas.android.com/tools"
+    tools:ignore="InsecureBaseConfiguration">
+    <base-config cleartextTrafficPermitted="true">
```

Comment:
This should not be needed in production

Reply:
It's not in production, it will only be used for acceptance build types (see the acceptance directory in its path)

Reply:
Thank you for explaining